CI/Spinx: sphinxcontrib.jquery  extension missing

When pushing a PR with documentation, I get the following error during teh CI phase. It complains, that sphinxcontrib.jquery is missing. I have the same problem when trying to build the documentation locally on my Debian bookworm system, and I cannot find/install the missing extension.
```
cd ../tools && python3 gen-cpydiff.py
sphinx-build -E -b html -d build//doctrees  -W --keep-going -j auto . build//html
Running Sphinx v7.2.6

Extension error:
Could not import extension sphinxcontrib.jquery (exception: No module named 'sphinxcontrib.jquery')
make: Leaving directory '/home/runner/work/micropython/micropython/docs'
make: *** [Makefile:67: html] Error 2
```

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -17,6 +17,6 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v4
     - name: Install Python packages
-      run: pip install Sphinx
+      run: pip install -r docs/requirements.txt
     - name: Build docs
       run: make -C docs/ html

--- a/docs/develop/gettingstarted.rst
+++ b/docs/develop/gettingstarted.rst
@@ -245,7 +245,7 @@ that you use a virtual environment:
 
    $ python3 -m venv env
    $ source env/bin/activate
-   $ pip install sphinx
+   $ pip install -r docs/requirements.txt
 
 Navigate to the ``docs`` directory:
 

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,0 +1,2 @@
+sphinx~=7.2.6
+sphinxcontrib.jquery==4.1

